pages/updates/index.vue

```diff
@@ -0,0 +1,761 @@
+<template>
+	<main>
+		<article id="5e3ea028-0697-4f95-89e4-0ed44d5f8b15" class="page sans">
+			<header>
+				<div class="page-header-icon undefined">
+					<span class="icon">📸</span>
+				</div>
+				<h1 class="page-title">Photography Lounge Updates</h1>
+			</header>
+			<div class="page-body">
+				<table class="collection-content">
+					<thead>
+						<tr>
+							<th>
+								<span class="icon property-icon"
+									><svg
```

Comment:
I'd add the nuxt svg loader and require the icons as components instead of inlining them here

Reply:
removed as of https://github.com/PhotographyLounge/site/pull/1/commits/820e56995adb66a071dee7d1fb49908194f8dd96